internal/core/command/utils.go

```diff
@@ -0,0 +1,28 @@
+package command
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/clients"
+)
+
+// addHeadersToRequest populates deviceServiceRequest with matching headers from
+// originalRequest to retain information that can be used by the device service.
+func addHeadersToRequest(originalRequest *http.Request, deviceServiceRequest *http.Request, context context.Context) {
+
+	var propagatedHeader = []string{clients.ContentType}
```

Comment:
Can we move this outside of this function so it acts more like a constant? Then we can comment what it is used for and future developers can simply just add or remove from this slice to get the desired affect.